### DISCUSSION TITLE
First Asteroid-based VAD

### DISCUSSION THREAD

**Discussion by mpariente** [asteroid-team/asteroid#577] [General]
## Highlights
- New ConvTasNet-based VAD on the [`Libri_VAD`]() dataset :rocket: 
- Fixed and improved GEVD beamformer
- Multi-Decoder DPRNN recipe

### Added
- [src&egs] Add VAD in Asteroid :tada: (#558)
- [src&tests] Add GEVD beamformer (#520)
- [hub] Add library version and name for Hugging Face download stats (#524)
- [egs] Add recipe for Multi-Decoder DPRNN (#463)
- [egs] Enable WER eval with GPU (#541)
- [egs] Add readme and pretrained model to FaSNet recipe (#561)

### Changed
- [src] Make cLN default in causal ConvTasNet (#511)
- [install] Pin pytorch-optimizer version to support RAdam (#568)

### Fixed
- [src] Prevent complex eigen values in gev (#519)
- [src] Fix default norm_type for causal ConvTasNet (#503)
- [egs] Fixed bug in X-UMX (#521)
- [nbs] asteroid.filterbanks to asteroid_filterbanks (#526)
- [nbs] Fix typos in notebooks/02_Filterbank.ipynb (#527)
- [install] Fix PL version <1.5.0 (#576)

Thanks to @ldelebec @hihunjin @nobel861017 @ben-freist @r-sawata @osanseviero @JunzheJosephZhu and @JorisCos  for their awesome contributions :fire: :muscle: :pray: 

<hr /><em>This discussion was created from the release <a href='https://github.com/asteroid-team/asteroid/releases/tag/v0.5.2'>First Asteroid-based VAD</a>.</em>

**Comment by hihunjin** [comment]
Great experience on contrubuting an awesome repo.
